README.md

```diff
@@ -367,6 +368,14 @@ Daily Builds
 [opensuse-42.1-version-1.1.X]: https://dotnetcli.blob.core.windows.net/dotnet/release/1.1.0/dnvm/latest.sharedfx.opensuse.42.1.x64.version
 [opensuse-42.1-targz-1.1.X]: https://dotnetcli.blob.core.windows.net/dotnet/release/1.1.0/Binaries/Latest/dotnet-opensuse.42.1-x64.latest.tar.gz
 
+[rhel-6-badge-master]: https://dotnetcli.blob.core.windows.net/dotnet/Runtime/release/master/sharedfx_rhel.6-x64_Release_version_badge.svg
```

Comment:
There is no `release/master` URL.  It is just `master`.  Repeated for all 3 links.

`https://dotnetcli.blob.core.windows.net/dotnet/Runtime/release/master/`
vs
`https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/`